docs/docs/http.md

```diff
@@ -72,3 +73,64 @@ curl http://10.190.238.173:8123/v1/services/_nginx._tcp.marathon.mesos.
 ]
 ```
 
+## `GET /v1/enumerate`
+
+Lists in JSON format all DNS information.
+
+```console
+curl http://master.mesos:8123/v1/enumerate
+{
+  "frameworks": [
+   {
+    "tasks": null,
```

Comment:
probably fine to have copied. omitting the `null` fields is something that should be configured at in the Golang source.